sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ClaimsBasedSecurityChannel.java

```diff
@@ -37,7 +45,7 @@ public ClaimsBasedSecurityChannel(Mono<RequestResponseChannel> responseChannelMo
         this.authorizationType = Objects.requireNonNull(authorizationType, "'authorizationType' cannot be null.");
         this.retryOptions = Objects.requireNonNull(retryOptions, "'retryOptions' cannot be null.");
         this.credential = Objects.requireNonNull(tokenCredential, "'tokenCredential' cannot be null.");
-        this.cbsChannelMono = Objects.requireNonNull(responseChannelMono, "'responseChannelMono' cannot be null.");
+        this.cbsChannelMono = Objects.requireNonNull(responseChannelMono, "'cbsChannelMono' cannot be null.");
```

Comment:
This didn't have to change. The error message should use the name of the param instead of the instance variable name.

Reply:
Good catch. I didn't check the actual parameter name.